Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -184,6 +184,40 @@
             "helpMarkDown": "Select the option to enable msdeploy flag MSDEPLOY_RENAME_FILES_FLAG in Azure App Service application settings. The option if set enables msdeploy to rename locked files that are locked during app deployment"
         },
         {
+            "name": "ScriptType",
```

Comment:
Should this script type input visibility controlled by an input for example: Run Script After Deploy or something?